### PR TITLE
Add defaut database seed for CC data

### LINKED ISSUE #27: Database Seeding
- Hinzufügen von Default Daten für Posten/Status
- Hinzufügen von zufällig generierten Personen etc. zum testen

### PR DESCRIPTION
**Pull request by olohr**
closes #27 
Only contains initial CC data for now, random data is next

### CODE REVIEW

**Comment by olohr** [comment]
There are probably smarter ways to generate random names, but it's good enough for testing

**Review by Yorreck — CHANGES_REQUESTED** [review]
Please fix the bug, that appears while trying to automatically delete a person linked to an account. just dont delete this person and maybe even not their references.

File: appsettings.json [hunk 1]
```diff
@@ -17,5 +17,17 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "DefaultConnection": "server=localhost;port=3306;database=personaltool;user=personaltool;password=personaltool;"
+  },
+  "RandomTestSeedConfig": {
+    "Enabled": false,
+    "ClearExistingData": false,
+    "Seed": 123,
```
Comment (Yorreck) [inline appsettings.json]: setting seed to null should lead to a random seed

**Review by Yorreck — CHANGES_REQUESTED** [review]

**Review by Yorreck — CHANGES_REQUESTED** [review]
I setup my config like this:
"RandomTestSeedConfig": {
    "Enabled": true,
    "ClearExistingData": true,
    "Seed": 123,
    "Persons": 50,
    "Positions": 0,
    "CareerLevels": 0,
    "MemberStatus": 0,
    "PersonPositions": 30,
    "PersonCareerLevels": 150,
    "PersonMemberStatus": 150
  }

Goal was to create the default entries for status etc. an create just random persons and links. Throwing an exception cause it cant create links between person and status or else.

File: appsettings.json (hunk 1 above)
Comment (olohr) [reply appsettings.json]: A null seed leads to a random seed, that's already implemented, or do you mean it should be the default? In that case, I agree

**Review by Yorreck — APPROVED** [review]

**Pull request merged**